TwitchPlaysAssembly/Src/Votes.cs

```diff
@@ -66,33 +66,39 @@ public static class Votes
 			VoteTypes.Solve, new VoteData {
 				validityChecks = new List<Tuple<Func<bool>, string>>
 				{
-					CreateCheck(() => !TwitchPlaySettings.data.EnableVoteSolve, "Sorry, {0}, votesolving is disabled."),
-					CreateCheck(() => voteModule.Votesolving, "Sorry, {0}, that module is already being votesolved."),
-					CreateCheck(() => OtherModes.currentMode == TwitchPlaysMode.VS, "Sorry, {0}, votesolving is disabled during vsmode bombs."),
-					CreateCheck(() => TwitchGame.Instance.VoteSolveCount >= 2, "Sorry, {0}, two votesolves have already been used. Another one cannot be started."),
+					CreateCheck(() => !TwitchPlaySettings.data.EnableVoteSolve, "Sorry, {0}, VoteSolving is disabled."),

```

Comment:
Why did you make votesolve capitalized? I don't think it should be capitalized.

Reply:
mainly a style choice, I can change it back if wanted